README.md

```diff
@@ -302,8 +302,7 @@ If the error is not closed and another error occurs, all errors will be listed a
 
 1. When showing the labels/names of the nodes, there is always one node that turns black.
 1. When using the input fields, the autocompletion does not add the selected word at the right place, when adding something in the middle.
-1. When data is obtained via a Cypher query, some links are not visible. When clicking on the eye-icon, connected
-   nodes are visible but the links are not. You can solve it by clicking on the `Add/show connecting nodes` button.
+1. When data is obtained via a Cypher query, some links are not visible. When clicking on the eye-icon, connected nodes are visible but the links are not. You can solve it by clicking on the `Add/show connecting nodes` button.
 
```

Comment:
GitHub rendered it just fine before this change of yours. Why did you change it? Long line lengths are not very nice to review and maintain.

![image](https://github.com/melexis/traceability-viewer/assets/28319872/539923d8-2764-4500-8f20-26992019932d)

Reply:
I removed the newline in the middle of the sentence because it would result in this:
![image](https://github.com/melexis/traceability-viewer/assets/62558419/61ac8469-8344-410f-a927-7e92fc40747d)

But I can add newlines after a sentence?

Reply:
I think I resolved it in 30a2aedf809fbcfa11292f6d77f54dfaec8458c7

Reply:
This syntax seems to work in Markdown. FYI In reStructuredText, you would need to align the text:
```
- Bullets are "-", "*" or "+".
  Continuing text must be aligned
  after the bullet and whitespace
  ```
  https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists